docs/0_internal/5_b_2_approve_whitelist.md

```diff
@@ -0,0 +1,37 @@
+# Approve the Whitelist Transaction
+
+Once the transaction is proposed, the multisig owners must approve the transaction.
+
+## Approve the Whitelist for registryCoordinator
```

Comment:
```suggestion
## Approve the Whitelist for RegistryCoordinator
```

Reply:
In the contract deployment config `registryCoordinator` is not capitalized. I think making it true to the config would help dissuade confusion.
![Uploading Screenshot 2024-11-29 at 14.23.52.png…]()